plugins/auth-backend/src/providers/github/provider.ts

```diff
@@ -107,9 +110,12 @@ export class GithubAuthProvider implements OAuthHandlers {
   }
 
   async start(req: OAuthStartRequest): Promise<RedirectInfo> {
+    const state = this.extraState
+      ? { extraState: this.extraState, ...req.state }
```

Comment:
If extraState was a hash of strings, you could do:

const state = { ...this.extraState, ...req.state }

Reply:
Nice. I'll come up with something